wandb/sdk/interface/interface.py

```diff
@@ -476,6 +476,27 @@ def _communicate_artifact(
     ) -> MessageFuture:
         raise NotImplementedError
 
+    def communicate_download_artifact(
```

Comment:
should you already use the correct verb of deliver instead of communicate?

Reply:
I have another PR out to make that update: https://github.com/wandb/wandb/pull/6531

Just want to test that properly to make sure i didn't miss anything

Reply:
ok, so please merge that change as well once you are done, we are trying to get rid of communicate and adding it back is a tech debt for us.